pkg/controller/rpaasinstance/rpaasinstance_controller.go

```diff
@@ -404,6 +434,177 @@ func (r *ReconcileRpaasInstance) reconcileNginx(nginx *nginxv1alpha1.Nginx) erro
 	return err
 }
 
+func (r *ReconcileRpaasInstance) reconcileCacheHeaterCronJob(instance *v1alpha1.RpaasInstance, plan *v1alpha1.RpaasPlan) error {
+	cronName := instance.Name + "-heater-cron-job"
+	cronJob := &batchv1beta1.CronJob{}
+
+	err := r.client.Get(context.TODO(), types.NamespacedName{Name: cronName, Namespace: instance.Namespace}, cronJob)
+	isNotFound := k8sErrors.IsNotFound(err)
+	if err != nil && !isNotFound {
+		return err
+	} else if !isNotFound {
+		logrus.Infof("CronJob %s is found, skipping creation", cronName)
+		return nil
+	}
```

Comment:
It took me a while to understand theses conditions. What do you think about changing it for clarity to something like:
```suggestion
	if err == nil {
		logrus.Infof("CronJob %s is found, skipping creation", cronName)
		return nil
	}
	if !k8sErrors.IsNotFound(err) {
		return err
	}
```